### DISCUSSION TITLE
Web Scrapting with Xpath - How to get full text from linked article?

### DISCUSSION THREAD

**Discussion by kabukimandelux** [FreshRSS/FreshRSS#4858] [Q&A] [answered]
Dear all,

Is there a way to get the full article content from the following blog [https://www.kaffeemacher.ch/blog/](url) ?
I get the article listing using:
![image](https://user-images.githubusercontent.com/18744493/202130451-c452fc0c-d5a2-4dcf-b339-b4314a54279b.png)

Is there any way to follow the linked url and get the `//div[@class="thecontent"]` from the full article page in order to read the article offline afterwards?

Thanks a lot!

**Comment by Frenzie** [comment, marked as answer]
Their feed at https://www.kaffeemacher.ch/blog/rss looks fairly complete to me?

**Reply by kabukimandelux** [reply]
Thanks again, don't know why I couldn't find the feed, but as you said it works perfectly!

**Comment by kabukimandelux** [comment]
Thanks will look at it tonight. Didn't see the rss feed in the source code.

**Reply by Frenzie** [reply]
It's not there, but on anything that looks like a blog there's a pretty decent chance that a /rss, /feed, /atom, /feed/rss, /feed/atom or something along those lines exists.

That does raise the question whether those should simply be tried by default for convenience. The answer isn't simple. On the one hand it's public and public is fair game, but on the other hand it wasn't explicitly referred to.
